README.md

```diff
@@ -14,7 +14,7 @@ the following:
 
 Once you have Python installed through Anaconda, do the following:
 
-    pip install -e git+https://github.com/juangpc/fieldline-opm-client.git#egg=fieldline_client
+    pip install git+https://github.com/juangpc/fieldline-opm-client.git#egg=fieldline_client -r https://raw.githubusercontent.com/juangpc/fieldline-opm-client/main/requirements.txt
```

Comment:
Try doing:

```py
pip install git+https://github.com/jasmainak/fieldline-opm-client.git#egg=fieldline_client -r https://raw.githubusercontent.com/jasmainak/fieldline-opm-client/main/requirements.txt
```

to test. I mirrored my `main` branch to be what it would look like if you merged this pull request